app/models/conference.rb

```diff
@@ -15,4 +17,15 @@ def tickets_left
     confirmed_attendances = attendances.select { |attendance| attendance.confirmed }
     tickets.to_i - confirmed_attendances.size
   end
+
+  private
+
+    def date_interval
+      if [starts_on, ends_on].count(&:nil?) == 1
+        errors.add(:base, 'Set both dates or none')
+      elsif starts_on.to_s > ends_on.to_s
```

Comment:
O, wait. I am not sure if that is what I meant. 
I suppose you mean:  `[starts_on, ends_on].all?` (ie "both dates are set, none is nil").

I'll wait for your answer in the previous comment, to be sure that we are aiming for the same thing. 🎯